api/src/identity-access-management/domain/usecases/authenticate-for-saml.usecase.js

```diff
@@ -78,8 +78,10 @@ async function authenticateForSaml({
 
     return token;
   } catch (error) {
-    if (error instanceof UserNotFoundError || error instanceof PasswordNotMatching) {
+    if (error instanceof UserNotFoundError) {
       throw new MissingOrInvalidCredentialsError();
+    } else if (error instanceof PasswordNotMatching) {
+      throw new MissingOrInvalidCredentialsError(error.meta);
```

Comment:
En retournant une meta dans le cas d'un password not matching, on montre au monde extérieur l'existence du compte. Une idée serait de n'ajouter la meta qu'après N tentatives de plus ?

Reply:
@Libouk @lego-technix 

C'est le but de la condition, qui ne renvoie le `remainingAttempts` que sur les 10 dernières tentatives (qui sont nécessaires pour afficher le décompte), sinon il sera null.
```js
const remainingAttempts =
        userLogin.remainingAttempts <= REMAINING_ATTEMPTS_THRESHOLD ? userLogin.remainingAttempts : null;
```

Il faudrait rendre paramétrable cette propriété, mais je ne vois pas comment faire autrement étant donné que j'ai besoin de cette info pour l'afficher.